projects/batfish/src/main/antlr4/org/batfish/grammar/cisco/Cisco_routemap.g4

```diff
@@ -68,6 +68,11 @@ boolean_as_path_in_rp_stanza
    AS_PATH IN expr = as_path_set_expr
 ;
 
+boolean_as_path_is_local_rp_stanza
```

Comment:
Needs handling in `CiscoControlPlaneExtractor and all the way through pipeline.
```java
private RoutePolicyBoolean toRoutePolicyBoolean(Boolean_simple_rp_stanzaContext ctx)
```

Reply:
done